component/DateSelectionMonthlyCalendar.js

```diff
@@ -1,5 +1,6 @@
 import {useStoreon} from 'storeon/preact'
-import {html, createNextFreeSlotsForDateKey as __, createNextFreeSlotsForDateKey} from '../helper/index.js'
+import {html, createNextFreeSlotsForDateKey} from '../helper/index.js'
```

Comment:
```suggestion
import {html, createNextFreeSlotsForDateKey, translator as __} from '../helper/index.js'
```
 és akkor alatta lévő sor nem kell